mgmt_cli_test.py

```diff
@@ -370,6 +372,30 @@ def test_manager_sanity(self):
             # Since this test activates encryption, it has to be the last test in the sanity
             self.test_client_encryption()
 
+    def test_manager_sanity_vnodes_tablets_cluster(self):
+        """
+        Test steps:
+        1) Create tablets keyspace and propagate some data.
+        2) Create vnodes keyspace and propagate some data.
+        3) Run sanity test (test_manager_sanity).
+        """
+        self.log.info('starting test_manager_sanity_vnodes_tablets_cluster')
+
+        tablets_keyspace = "tablets_keyspace"
+        self.create_keyspace(tablets_keyspace, replication_factor=3)
```

Comment:
I'd be more explicit here and also added `TabletsConfiguration` so in case tablets are not turned on it should fail.

Reply:
Fixed